_docs/_latest/configure/notifier/index.md

```diff
@@ -2,4 +2,140 @@
 title: Notifier
 order: 300
 ---
+
 # {{ page.title }}
+
+Forseti Notifier can send a variety of notifications to alert you
+of Forseti events. These notifications can be configured to be sent
+to different channels, and in different formats.
+
+### Notification Types
+
+  1. Inventory summary
+  1. Scanner summary
+  1. Violations
+  1. CSCC findings
+  
+### Notification Channels
+
+  1. Email
+  1. Slack
+  1. GCS
+
+### Notification Formats
+
+  1. csv (human-readable data)
+  1. json (structured data)
+
+## Configuring Notifier
+
+### Inventory Summary (TBD)
+
+### Scanner Summary (TBD)
+
+### Violation Notifications
+
```

Comment:
Add an intro sentence, something like: "To select how you want Notifier to send notification violations, follow the steps below:"

Reply:
Done, thanks!